flink-ml-benchmark/src/main/java/org/apache/flink/ml/benchmark/Benchmark.java

```diff
@@ -31,34 +31,33 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.List;
+import java.util.Collections;
 import java.util.Map;
 
 /** Entry class for benchmark execution. */
 public class Benchmark {
-    private static final Logger LOG = LoggerFactory.getLogger(Benchmark.class);
-
     static final String VERSION_KEY = "version";
 
-    static final Option HELP_OPTION =
+    private static final Logger LOG = LoggerFactory.getLogger(Benchmark.class);
+
+    private static final Option HELP_OPTION =
             Option.builder("h")
                     .longOpt("help")
                     .desc("Show the help message for the command line interface.")
                     .build();
 
-    static final Option OUTPUT_FILE_OPTION =
+    private static final Option OUTPUT_FILE_OPTION =
             Option.builder()
                     .longOpt("output-file")
                     .desc("The output file name to save benchmark results.")
                     .hasArg()
                     .build();
 
-    static final Options OPTIONS =
+    private static final Options OPTIONS =
             new Options().addOption(HELP_OPTION).addOption(OUTPUT_FILE_OPTION);
 
-    public static void printHelp() {
+    private static void printHelp() {
         HelpFormatter formatter = new HelpFormatter();
         formatter.setLeftPadding(5);
         formatter.setWidth(80);
```

Comment:
Should we update the line below to use the new script name?

And we can simplify the code a bit by using `\n` in `"./benchmark-run.sh <config-file-path> [OPTIONS]\n"` instead of using `System.out.println()`.